### ISSUE TITLE
Screen reader is not announcing the updated data information in the table on pressing the 'ID' column header in Windows.

### ISSUE THREAD

**Issue by msftedad** [tonytomov/jqGrid#1078]
**Test Environment:**
OS: Windows 11 Version 24H2(OS Build 26120.1930)
Browser: Microsoft edge 135.0.3179.98
URL: [js-tutorials.com : https://free-jqgrid.github.io/getting-started/index.html#the_first_grid
Screen reader: JAWS 2025.2504.89, NVDA 2025.2412.50.400

**User Impact:**
Screen reader dependent users will not know whether their action is performed or not if it doesn't announce any information.

**Observation:**
This issue is not observed with Narrator

**Pre-Requisite:**
Turn on Screen Readers:
NVDA: Ctrl + Alt + N
JAWS: Ctrl + Alt + J
Verbosity:
JAWS: Beginner, Highest
NVDA: Default

**Repro steps:**
1. Open the URL: https://free-jqgrid.github.io/getting-started/index.html#the_first_grid in the latest edge browser.
2. Demo page will appear.
3. Press tab key to reach to the 'ID' control of the table and press enter to activate it.
4. Verify whether screen reader is announcing the updated data information in the table on pressing the 'ID' column header or not.

**Actual behavior:**
On pressing enter key to activate the 'ID' column header screen reader doesn't announce any information about the updated data in the table in Windows.

**Please Refer attachment:** 
1. Screen reader is not announcing the updated data information in the table on pressing the 'ID' column header in Windows.mp4
2. Screen reader is not announcing the updated data information in the table on pressing the 'ID' column header in Windows.png

https://github.com/user-attachments/assets/76433963-37c5-4dc4-96e6-8ba1516ab056
<img width="960" height="558" alt="Image" src="https://github.com/user-attachments/assets/5418756f-564c-4424-a659-5b60e764e0c9" />

**Expected behavior:**
On pressing enter key to activate the 'ID' column header screen reader should announce the information about the updated data in the table in Windows.
